crates/api/proto/gen/kitsune2.fetch.rs

```diff
@@ -1,6 +1,6 @@
 // This file is @generated by prost-build.
 /// A fetch request.
-#[derive(Clone, PartialEq, ::prost::Message)]
+#[derive(Clone, PartialEq, Eq, Hash, ::prost::Message)]
```

Comment:
Interesting that this got added now, I don't know why.

Reply:
Probably because it's a new version of the library and we haven't regenerated the Rust code. It was on the TODO list at some point to check that these generated files are up to date on CI. Not sure if that ever made it to an issue.